app/src/main/java/com/andrew/tasky/agenda/data/reminder/ReminderRepositoryImpl.kt

```diff
@@ -111,12 +141,20 @@ class ReminderRepositoryImpl @Inject constructor(
         }
     }
 
+    override suspend fun completeReminderNotification(reminderId: String) {
+        val reminder = db.getReminderDao().getReminderById(
+            reminderId
+        )?.copy(notificationCompleted = true)
```

Comment:
Also this shouldn't be necessary since you know that a notification must be completed if `current time > agenda item time - remind at time`